src/lib/gundb/UserStorageClass.js

```diff
@@ -842,14 +850,7 @@ export class UserStorage {
    * @returns {object} feed item or null if it doesn't exist
    */
   getFeedItemByTransactionHash(transactionHash: string): Promise<FeedEvent> {
-    return this.feed
```

Comment:
i think better to keep old implementation (+ save feed to cache once it's loaded) as the fallback is something went wrong and cache suddenly turned out to be empty for specific feed

Reply:
explain... what do you mean? what fallback?

Reply:
you're just returning this.feedIds[transactionHash]
what if this.feedIds[transactionHash] is empty or undefined (no transactionHash in this.feedIds object) ?

we need to have some fallback in this case

e.g.

```javascript
const feedItem = this.feedIds[transactionHash]

if (feedItem) {
  return feedItem
}

return this.feed
  .get('byid')
  .get(transactionHash)
  .decrypt()
  .then(feedItem => {
    this.feedIds[transactionHash] = feedItem
    // update feed cache here
  })
  .catch(e => {
    // log error here
    return undefined
  })
```

Reply:
ok added